nx-X11/lib/configure.ac

```diff
@@ -0,0 +1,283 @@
+# Initialize Autoconf
+AC_PREREQ([2.60])
+AC_INIT([libNX_X11], [1.6.5],
+        [https://github.com/ArcticaProject/nx-libs/issues], [libNX_X11])
+AC_CONFIG_AUX_DIR([build-aux])
+AC_CONFIG_SRCDIR([Makefile.am])
+AC_CONFIG_HEADERS([src/config.h include/X11/XlibConf.h])
+AC_CONFIG_MACRO_DIR([m4])
+
+AC_CANONICAL_BUILD
+AC_CANONICAL_HOST
+
+# Set common system defines for POSIX extensions, such as _GNU_SOURCE
+# Must be called before any macros that run the compiler (like AC_PROG_LIBTOOL)
+# to avoid autoconf errors.
+AC_USE_SYSTEM_EXTENSIONS
+
+# Initialize Automake
+AM_INIT_AUTOMAKE([foreign dist-bzip2])
+
+# Initialize libtool
+AC_PROG_LIBTOOL
+
+# Upstream's pkg.m4 (since 0.27) offers this now, but define our own
+# compatible version in case the local version of pkgconfig isn't new enough.
+# https://bugs.freedesktop.org/show_bug.cgi?id=48743
+m4_ifdef([PKG_INSTALLDIR], [PKG_INSTALLDIR],
+         [AC_ARG_WITH([pkgconfigdir],
+                      [AS_HELP_STRING([--with-pkgconfigdir],
+                      [install directory for nxcompshad.pc pkg-config file])],
+                       [],[with_pkgconfigdir='$(libdir)/pkgconfig'])
+          AC_SUBST([pkgconfigdir], [${with_pkgconfigdir}])])
+
+NX_COMPILER_BRAND
+NX_DEFAULT_OPTIONS
+
+# Checks for pkg-config packages
+
+X11_REQUIRES='nx-xproto nx-xextproto nx-xtrans'
+X11_EXTRA_DEPS=""
+
+# Issue an error if nx-xtrans.m4 was not found and NX_XTRANS_CONNECTION_FLAGS macro
+# was not expanded, since libNX_X11 with no transport types is rather useless.
```

Comment:
>> Why did you leave out the "If you're seeing an error here ..." comment?

> Good catch. C+P flaw. Re-added.

Ah, no... In fact, I dropped it on purpose. Because it refers to the separate shipping of Xtrans and libX11 in X.org. They are different repos. In NX_X11, we have the files in one repo. Even if we dissect nx-libs and split it up into nxcomp, nxcompshad, NX_X11 and nxagent and nxproxy, the Xtrans files will still stay in the NX_X11 part. So the location of those files is always well-defined and the NX_Xtrans files cannot be installed separately.

Thus, doing nothing here...